sklearn/feature_selection/_rfe.py

```diff
@@ -674,6 +680,20 @@ class RFECV(RFE):
 
             .. versionadded:: 1.5
 
+        split(k)_ranking : ndarray of shape (n_subsets_of_features,)
+            The cross-validation rankings across (k)th fold.
+            Selected (i.e., estimated best) features are assigned rank 1.
+            Illustration in
+            :ref:`sphx_glr_auto_examples_feature_selection_plot_rfe_with_cross_validation.py`
```

Comment:
shouldn't we then refine the example to include this?

Reply:
I added an exemple about how to use the support.